doc/nitpick_ignore

```diff
@@ -28,6 +28,8 @@ py:class scipy.spatial.transform.Rotation
 # derived from scipy
 py:method weldx.transformations.WXRotation.from_mrp
 py:method weldx.transformations.WXRotation.random
+py:obj weldx.transformations.WXRotation.from_mrp
```

Comment:
what do I have to put here to get rid of the following warning @vhirtham ?
`:docstring of weldx.transformations.WXRotation.from_mrp:12: WARNING: py:obj reference target not found: mrp[i]`

Reply:
usually exact the same thing that is written after the `:`

Reply:
nice, thanks !